### DISCUSSION TITLE
<title>React Redux

### DISCUSSION THREAD

**Discussion by muhammadrahim124** [community/community#102702] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

what is React Redux

**Comment by Mshahzad56** [comment]
React Redux is the official React UI bindings layer for Redux. It lets your React components read data from a Redux store, and dispatch actions to the store to update state.

**Comment by ettaboyle** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
